Src/BootCamp.Chapter/TextTable.cs

```diff
@@ -38,7 +38,87 @@ public static class TextTable
         /// </summary>
         public static string Build(string message, int padding)
         {
-            return "";
+            if (message.Length == 0)
+            {
+                return "";
+            }
+            // make a array so I can find the longest word. 
+
+            var arrayOfLines = message.Split(Environment.NewLine);
+
+            // Find length of longest word 
+            var lengthLongestWord = 0;
+
+            if (arrayOfLines.Length == 1)
+            {
+                lengthLongestWord = arrayOfLines[0].Length;
+            }
+            else
+            {
+                foreach (var word in arrayOfLines)
+                {
+                    if (word.Length > lengthLongestWord)
+                    {
+                        lengthLongestWord = word.Length;
+                    }
+                }
+            }
```

Comment:
This could have been a function called "GetLongestWordLength" or something. But I think it's fine.

You could write
```csharp
var lengthLongestWord = 0;
foreach (var word in arrayOfLines)
{
    if (word.Length > lengthLongestWord)
    {
        lengthLongestWord = word.Length;
    }
}
```
and the functionality would be the same. :)

Reply:
Thanks for the repyly. I still going to change it.